### DISCUSSION TITLE
"Move" a project in a studio to first position

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6561] [Ideas]
I create studios for my students where I collect projects for specific topics. I keep adding new projects and every new project always gets by default the first position in the studio. Sometimes, even adding new projects, I would like to keep a project or several projects in the first positions. So, after I add the new project, I remove from the studio the project that I want in first position and then I add it back. I have to delete the project first, as projects that are already in a studio don't get added again. 

My proposal, when you add a project to a studio, is to remove the project first, so after adding it it will always be in first position, even if it was already in the studio.

**Comment by WorldLanguages** [comment]
I guess you want to "pin" a few projects in a studio?

So after adding a new project, Scratch Addons would do this:
1. Remove the pinned projects
2. Add the project you want to add
3. Readd the pinned projects in reverse order (so that they are visibly in the right order)

**Comment by sfederici** [comment]
That would be an alternative solution
